libvcx/src/abi_utils/mod.rs

```diff
@@ -0,0 +1,28 @@
+use crate::aries::handlers::connection::connection::ConnectionState;
```

Comment:
How about moving this code to `src/api/mod.rs`, or moving some of the enum conversion stuff to this module, along with `src/api/return_types_u32.rs`?

Reply:
happy to do that change

Reply:
Well I played a bit and took this suggestion few more steps ahead - I turned it into separate PR https://github.com/hyperledger/aries-vcx/pull/295/files so the original purpose of this one does not get too polluted

Reply:
That's perfect, thank you